src/orchestrator.py

```
@@ -0,0 +1,247 @@
+            
+            print(f"\nFound {len(tests)} test(s) to execute")
+            
+            # Run each test and collect report filenames
+            report_files = []
+            for test_name, test_config in tests.items():
+                try:
+                    test_image = test_config.get('image')
+                    test_config_values = test_config.get('config', {})
+                    report_file = self.run_test(test_name, test_image, test_config_values)
+                    report_files.append(report_file)
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)